apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -4368,6 +4369,27 @@ sophia_crypto(_Cfg) ->
                                 , {test_string_verify_secp256k1, Message, true}
                                 , {test_string_verify_secp256k1, <<"Not the secret message">>, false}] ],
 
+    %% Test ecrecover
+    %%   These static examples are taken from
+    %%   https://github.com/aeternity/parity-ethereum/blob/master/ethcore/builtin/src/lib.rs#L656
```

Comment:
It would be nice to check that we can actually sign things and get it through ecrecover (like we do for ecverify). I.e. start from a message and a keypair?!

Reply:
Ok, then I'll extend that.